content/actions/learn-github-actions/creating-workflow-templates.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Creating workflow templates
-shortTitle: Creating templates
-intro: Learn how you can create workflow templates to help people in your team add new workflows more easily.
+title: Creating starter workflows
```

Comment:
@pulkitaggarwl you will need to rename file as well to creating-starter-workflows.md